docker-images/cts-base/README

```diff
@@ -54,3 +54,23 @@ Build command : docker build -t jakartaee/cts-base:0.3 -f Dockerfile3 .
 Image push command : docker push jakartaee/cts-base:0.3
 
 =====================================
+
+Dockerfile4:
+
+Image - jakartaee/cts-base:0.4
```

Comment:
We should wait until cts-base:0.4 is uploaded to the Docker repository before merge this change.  Otherwise this pr looks good to me.

Reply:
Yeah, also I would wait yet to check once again that all changes in 10.0.x branch are alright, so if there would be something to fix, I would just update this PR with the fix.

Reply:
@scottmarlow I believe that now it is alright. If you approve the PR, I will merge it and test and fix all builds same as I did for 10.x. If anything breaks later, just ping me and I will fix it asap.